Sources/SnapshotPreviewsCore/View+Snapshot.swift

```diff
@@ -54,50 +54,102 @@ extension View {
     controller: ExpandingViewController,
     window: UIWindow,
     async: Bool,
-    completion: @escaping (SnapshotResult) -> Void)
-  {
+    completion: @escaping (SnapshotResult) -> Void
```

Comment:
When trying to test, I ran into a bunch of errors in the file, namely "Type of expression is ambiguous without a type annotation" a number of times.

These changes aren't final, more a test I figured, so I just resolved them with these changes. Not married to them at all